### PR TITLE
🎥 Video Component

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per @bettybondoc's figma model, https://www.figma.com/design/Jyj0SszjfSBc6Minkl3g1l/Tina.io?node-id=399-5400 we should have a video component. 

This PR creates a video component

Users have the option between
- [ ] YouTube embed URLs 
- [ ] Cloudinary Media (image or video) 
- [ ] Users can input custom thumbnail photos for Cloudinary media  

3849a9ff-aa1a-404f-bdba-ad1e62830cd3

**Figure: New 🎥 Component**

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
Looking pretty awesome! 

1. Work on responsiveness
2. Work on null-handling
3. work on branch spelling ;)

**Comment by joshbermanssw** [comment]
> Looking pretty awesome!
> 
> 1. Work on responsiveness
> 2. Work on null-handling
> 3. work on branch spelling ;)

Done except for brnch speling

File: components/blocks/MediaComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  const renderMedia = (media, index) => {
+    const isPlaying = isPlayingArray[index];
+
+    const sizeClasses = "w-[500px] h-[300px] sm:w-[500px] sm:h-[300px] md:w-[500px] md:h-[320px] lg:w-[600px] lg:h-[340px] xl:w-[600px] xl:h-[340px]";
+
+    if (
+      media.__typename ===
+      'PageBlocksMediaComponentMediaItemCloudinaryMediaComponent'
```
Comment (bradystroud) [inline components/blocks/MediaComponent.tsx:24]: Any ideas to make it so these are not hardcoded?

**Review by bradystroud — CHANGES_REQUESTED** [review]
Just one comment.

Also I'd love a clear designer 'test pass' on this before it gets merged

File: components/blocks/MediaComponent.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/blocks/MediaComponent.tsx:24]: not within the media data object

**Comment by bettybondoc** [comment]
✅ Test pass - I'm assuming we won't let the error to happen so I'm ignoring that LOL!

image

**Comment by joshbermanssw** [comment]
> ✅ Test pass - I'm assuming we won't let the error to happen so I'm ignoring that LOL!
> 
> image

thats just if you dont put thumbnail !

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**